### ISSUE TITLE
Support of IIMageProvider in PPTX Document

### ISSUE THREAD

**Issue by Youslam** [opensagres/xdocreport#196]
Is it possible to manage dynamic images for pptx document with IImageProvider or with file manager?

Thank you in advance

**Comment by angelozerr** [comment]
No sorry, no time to support that.

**Comment by Youslam** [comment]
Thank you for your response.
Can you give me tips because I want to add this feature.
What exactly is missing to implement this feature?

Thank you in advance

**Comment by angelozerr** [comment]
To be honnest with you, I think it's a very big hard task. I suggest you that you see how to docx support that.

Good luck!

**Comment by Youslam** [comment]
Yes I think so. I began to see the code. it seems to be very hard. But I will try as.
The first problem I encountered is that Power Point does not support bookmark.

Thank you
